pulp_2to3_migration/app/plugin/content.py

```diff
@@ -233,8 +233,7 @@ def get_remote_by_importer_id(importer_id):
             pulp_2to3_detail_qs = pulp_2to3_detail_qs.order_by('repo_id')
 
         with ProgressReport(
-            message='Migrating {} content to Pulp 3 {}'.format(self.migrator.pulp2_plugin,
-                                                               content_type),
+            message='Migrating {} content to Pulp 3'.format(content_type),
```

Comment:
Was `Migrating iso content to Pulp 3 iso`, `Migrating rpm content to Pulp 3 erratum`.
Now ` Migrating iso content to Pulp 3`, `Migrating erratum content to Pulp 3 `.